signer/web/src/providers/appProvider.js

```diff
@@ -14,14 +14,19 @@ export default class appProvider {
 
   onWsOpen () {
     logger.log('[APP Provider] connected');
-    this.store.dispatch(updateAppState({ isParityRunning: true, isLoading: false, isWsConnected: true }));
+    this.store.dispatch(
+      updateAppState({ isParityRunning: true, isLoading: false, isWsConnected: true })
+    );
   }
 
   onWsError () {
-    isParityRunning(this.wsPath)
-      .then(isRunning => {
-        this.store.dispatch(updateAppState({ isParityRunning: isRunning, isLoading: false, isWsConnected: false }));
-      });
+    this.store.dispatch(
+      updateAppState({ isLoading: false, isWsConnected: false })
+    );
+    // isParityRunning(this.wsPath)
```

Comment:
Or I'm pretty sure we already have a `ping` endpoint - it should be possible to check if parity is running or not.